### DISCUSSION TITLE
nonebot如何不重启地加载新的插件

### DISCUSSION THREAD

**Discussion by YangRucheng** [nonebot/discussions#251] [问答]
nonebot能否热加载插件? 
我在教程中没有看到这样的功能, 以后会考虑支持吗

**Comment by yanyongyu** [comment]
目前只有冷重载模式，在文件修改时自动重启，热重载模式由于Python的限制暂时不能很好的做到

**Comment by mnixry** [comment]
https://github.com/nonebot/nonebot2/issues/880
